test/bugs-reported/bug2.chpl

```diff
@@ -0,0 +1,102 @@
+
```

Comment:
I agree, the code is irrelevant to the issue being addressed here. Updated bug2.chpl, bug2.good, bug2.bad accordingly. Made the code more shorter and relevant.